sheets/1/exercise-3.tex

```diff
@@ -5,6 +5,24 @@
 In jeder teilaufgabe wir ein bestimmter teil von Java erklärt und du sollst ein Beispiel aus Aufgabe 1 finden. In \textbf{[]} steht immer wie viele verschiedenen Beispiele du finden solltest.
 
 \begin{enumerate}
+    \item \textbf{[2]} Bevor wir uns mit dem Java Programmcode ansich beschäftigen, sollten wir noch Wissen was kommentare sind.
+    Weil es kommt vor das wir dinge in die Java datei schreiben wollen, einfach als Notiz oder auch als Erklärung, für andere Menschen. Da der Computer sie nicht lesen soll, müssen wir sie im Markieren.
+          Für kurze Kommentare kann man \lstinline[breaklines=false]{//} verwenden. Dammit markiert man den Rest der Zeile als Kommentar. Beachte das manche Programme, automatisch ein Zeilen umbruch einfügen wenn die Zeile zu lang wird.
+          Das ist aber meistens nur Darstellung und gilt nicht als ende der Zeile.
+          Alternativ kann man den kommentar auch mit \lstinline[breaklines=false]{/* */} umschließen. Dadurch kann man auch ein mehrzeiligen kommentar schreiben. Dabei ist es konnvention, jede Zeile mit einem \textcolor{javagreen}{\texttt{*}} zu beginnen.
```

Comment:
```suggestion
          Alternativ kann man den kommentar auch mit \lstinline[breaklines=false]{/* */} umschließen. Dadurch kann man auch ein mehrzeiligen kommentar schreiben. Dabei ist es üblich jede Zeile mit einem \textcolor{javagreen}{\texttt{*}} zu beginnen (wie in dem Beispiel unten zu sehen).
```

Reply:
Der * gehört schon zu java code convention.
Ich habe aber den Satz in () angefügt.

Reply:
Üblich sollte das einfacher zu verstehende Wort sein. Deshalb habe ich das vorgeschlagen.